### DISCUSSION TITLE
字幕同步功能出现错误

### DISCUSSION THREAD

**Discussion by wathper** [dyphire/mpv-config#53] [General]
MKV视频中存在多个字幕轨道，选择与音频同步时，提示“字幕同步失败。
请确保在脚本配置文件中为 alass 指定了正确的路径。”
在autosubsync.conf中指定为绝对路径后，依旧提示“字幕同步失败。
请确保在脚本配置文件中为 alass 指定了正确的路径。”

**Comment by dyphire** [comment]
无法复现，如果没有使用 release 中的整合包的话，请参考整合包的整体文件结构使用脚本和 alass 程序

**Comment by dyphire** [comment]
有进一步的反馈吗？如果使用整合包依然出现此问题建议上传日志和录屏文件

**Comment by dyphire** [comment]
我设法重现了这个问题，并通过单独使用 alass 命令查看日志定位到了问题所在。这个是脚本提示错误，实际上是 alass 无法正确提取音轨文件导致的失败，可以参考 https://github.com/kaegi/alass/issues/24
鉴于 alass 开发者已不再维护，这个问题恐怕得不到解决，尝试使用脚本支持的另一个工具 [ffsubsync](https://github.com/smacke/ffsubsync) 来进行字幕同步吧

**Reply by wathper** [reply]
具体是什么问题我也不太清楚,多谢告知了。
